docs/pages/admin-guides/access-controls/sso/sso.mdx

```diff
@@ -615,6 +615,55 @@ If something is not working, we recommend to:
 
 - Double-check the host names, tokens and TCP ports in a connector definition.
 
+## Identifier-first login
+
+Identifier-first login is a feature that simplifies the login experience in clusters with
+multiple authentication connectors. When identifier-first login is enabled, the initial
+login screen prompts the user for their username. After entering their username, the 
+user is only presented with the connector(s) mapped to them.
+
+This feature can be enabled on a cluster by configuring any one or more SAML, OIDC, or
+GitHub auth connector with a `spec.user_matchers` field, which takes a set of one or more
+user matchers for usernames. A user matcher is a string matcher that can either be a specific
```

Comment:
I would add an example here since we are mentioning specific YAML fields. Otherwise, it takes some effort to visualize what's going on.